rules/cross-platform/defense_evasion_timestomp_touch.toml

```diff
@@ -33,10 +33,11 @@ query = '''
 process where event.type == "start" and
  process.name : "touch" and user.id != "0" and
  process.args : ("-r", "-t", "-a*","-m*") and
- not process.args : ("/usr/lib/go-*/bin/go", "/usr/lib/dracut/dracut-functions.sh", "/tmp/KSInstallAction.*/m/.patch/*")
+ not process.args : ("/usr/lib/go-*/bin/go", "/usr/lib/dracut/dracut-functions.sh", "/tmp/KSInstallAction.*/m/.patch/*") and
+ not process.command_line : "touch -a ../../../../ios/build/generated/ios/./FBReactNativeSpec/FBReactNativeSpec.h ../../../../ios/build/generated/ios/./FBReactNativeSpec/FBReactNativeSpec-generated.mm" and
```

Comment:
This seems too specific. Anything better to exclude on, like maybe the parent calling touch?